iree/compiler/Conversion/LinalgToSPIRV/BUILD

```diff
@@ -41,10 +41,10 @@ cc_library(
         "FoldGPUProcessorIDUses.cpp",
         "KernelDispatchUtils.cpp",
         "LinalgTileAndDistributePass.cpp",
-        "LinalgTileAndFusePass.cpp",
         "MatMulVectorizationTest.cpp",
         "Passes.cpp",
         "SplitDispatchFunctionPass.cpp",
+        "TileAndVectorizeInOneWorkgroupPass.cpp",
```

Comment:
Just make this `TileAndVectorizePass`. Why in one workgroup?

Reply:
It's emphasizing that his is not performing the first-level tiling. It is also a nice contrast to `ConcretizeTileAmongWorkgroupsPass`. :)

Reply:
I dont like the new name :D , but the one thing worse than API wars is naming wars, so I will not harp on this anymore.

Reply:
Hehe. Yeah I agree it's subjective. But it should be an improvement over the current name I believe. :D